vagrant/Vagrantfile

```diff
@@ -24,8 +24,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     t.vm.network :forwarded_port, guest: 8833, host: 8833
   end
   
-  # disable automatic update check:
-  config.vm.box_check_update = false
+  # enable automatic update check:
+  config.vm.box_check_update = true
```

Comment:
Why this change?

Reply:
I noticed that when debugging, why my vagrant setup didn't work, with this set= true, it just tells you, that you should update your vagrant buster image before (I had 9.99 and there is 10.0 out already). otherwise the apt-installation failed